### PR TITLE
Adding serverless operator 1.10

### PR DESCRIPTION
**Pull request by matzew**
Signed-off-by: Matthias Wessendorf <mwessend@redhat.com>

@mgencur please review.

This is based on the MASTER config;  

Not sure if we still need the 0.15 SRC references.

Also, not exactly sure on the `dependencies` thats a bit of a diff to 1.9

### CODE REVIEW

File: ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-release-1.10.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.13
```
Comment (markusthoemmes) [inline ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-release-1.10.yaml:5]: Let's make this 1.14.

Comment (matzew) [reply ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-release-1.10.yaml:5]: juchuuuuuu - good, thanks

I will bump (dif PR) for master too, than

**Comment by markusthoemmes** [comment]
This should rather be based on master I believe. The dependencies stuff is kinda important to make sure we only run against stable OCP versions.

**Comment by matzew** [comment]
> This should rather be based on master I believe. The dependencies stuff is kinda important to make sure we only run against stable OCP versions.

I did copy the master config, and turned into the 1.10 version config, I tried to say that in the description of the PR :-) 

Or do you mean differently?

**Review by markusthoemmes — APPROVED** [review]
/lgtm
/approve

**Pull request merged**